README.md

```diff
@@ -81,6 +81,21 @@ vars:
     quickbooks_union_databases: ['quickbooks_usa','quickbooks_canada'] # use this if the data is in different databases/projects but uses the same schema name
 ```
 
+### Customize the ordinal for account classes in `dbt_quickbooks` models 
+The current default numbering for ordinals is based on best practices for balance sheets and profit-and-loss statements in accounting. You can see these ordinals in action in the `quickbooks__general_ledger_by_period`, `quickbooks__balance_sheet` and `quickbooks__profit_and_loss` models. The ordinals are assigned off of the `account_class` values.
```

Comment:
It may be worthwhile to link to the [exact code snippet](https://github.com/fivetran/dbt_quickbooks/blob/f2dc731b23886c5e185aa390832f8f61e3725c37/models/quickbooks__general_ledger_by_period.sql#L44-L50) to show what the default is if customers are curious.